zerver/lib/actions.py

```diff
@@ -2591,6 +2601,21 @@ def gather_subscriptions_helper(user_profile):
         else:
             unsubscribed.append(stream_dict)
 
+    all_streams_id_set = set(all_streams_id)
+    never_subscribed_stream_ids = all_streams_id_set - sub_unsub_stream_ids
+    never_subscribed_streams = [ns_stream_dict for ns_stream_dict in all_streams if ns_stream_dict['id'] in never_subscribed_stream_ids]
+
+    for stream in never_subscribed_streams:
+        if not stream['invite_only']:
+            stream_dict = {'name': stream['name'],
+                           'invite_only': stream['invite_only'],
+                           'stream_id': stream['id'],
+                           'description': stream['description']}
```

Comment:
In this loop, stream is 

    {'name': u'Rome', 'email_token': u'525b13f2dd296cf839ae0182ec3bace1', 
    'realm__domain': u'zulip.com', 'invite_only': False, 'realm_id': 1, 'id': 1, 'description': u''}

Whereas, what we are adding to never_subscribed is 

    {'stream_id': 1, 'description': u'', 'name': u'Rome', 'invite_only': False}. 
So, how can I use `copy.deepcopy` here? Am I missing something?